### ISSUE TITLE
Error: System service failed

### ISSUE THREAD

**Issue by nfaltir** [getumbrel/umbrel#1370]
I am experiencing an issue. My Umbrel is not booting properly 😟, It keeps throwing this error `Error: System Service Failed`

I was able to take a peek inside `ssh -t umbrel@umbrel.local` to try and figure out the issue.

I found these errors in the log dir

```
ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements

electrs logs
--------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements

Tor logs
--------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)

================
==== Result ====
================
The debug script did not automatically detect any issues with your Umbrel.

```

Can anyone give me some tips on how to fix this issue?  🙏   I have tried the following below

- restarting
- made sure everything is plugged in correctly
- check power supply
- shutdown then restart

Any help is super appreciated! Thanks ☂️

**Comment by louneskmt** [comment]
Can you also share the services logs (at the beginning of the debug logs)?

**Comment by nfaltir** [comment]
```

Startup service logs
--------------------
-- Logs begin at Mon 2022-05-09 11:59:33 UTC, end at Mon 2022-05-09 12:10:07 UTC. --
May 09 11:59:45 umbrel systemd[1]: Dependency failed for Umbrel Startup Service.
May 09 11:59:45 umbrel systemd[1]: umbrel-startup.service: Job umbrel-startup.service/start failed with result 'dependency'.

External storage service logs
-----------------------------
-- Logs begin at Mon 2022-05-09 11:59:33 UTC, end at Mon 2022-05-09 12:10:07 UTC. --
May 09 11:59:37 umbrel systemd[1]: Starting External Storage Mounter...
May 09 11:59:37 umbrel external storage mounter[485]: Running external storage mount script...
May 09 11:59:37 umbrel external storage mounter[485]: No block devices found
May 09 11:59:37 umbrel external storage mounter[485]: Waiting for 5 seconds before checking again...
May 09 11:59:43 umbrel external storage mounter[485]: Found device "SanDisk Extreme 55AE"
May 09 11:59:43 umbrel external storage mounter[485]: Blacklisting USB device IDs against UAS driver...
May 09 11:59:43 umbrel external storage mounter[485]: Rebinding USB drivers...
May 09 11:59:43 umbrel external storage mounter[485]: Checking USB devices are back...
May 09 11:59:43 umbrel external storage mounter[485]: Waiting for USB devices...
May 09 11:59:44 umbrel external storage mounter[485]: Waiting for USB devices...
May 09 11:59:45 umbrel external storage mounter[485]: Checking if the device is ext4...
May 09 11:59:45 umbrel external storage mounter[485]: No, it's not ext4
May 09 11:59:45 umbrel external storage mounter[485]: Formatting device...
May 09 11:59:45 umbrel external storage mounter[485]: wipefs: error: /dev/sda: probing initialization failed: No medium found
May 09 11:59:45 umbrel systemd[1]: umbrel-external-storage.service: Main process exited, code=exited, status=1/FAILURE
May 09 11:59:45 umbrel systemd[1]: umbrel-external-storage.service: Failed with result 'exit-code'.
May 09 11:59:45 umbrel systemd[1]: Failed to start External Storage Mounter.

External storage SD card update service logs
--------------------------------------------
-- Logs begin at Mon 2022-05-09 11:59:33 UTC, end at Mon 2022-05-09 12:10:07 UTC. --
May 09 11:59:45 umbrel systemd[1]: Dependency failed for External Storage SDcard Updater.
May 09 11:59:45 umbrel systemd[1]: umbrel-external-storage-sdcard-update.service: Job umbrel-external-storage-sdcard-update.service/start failed with result 'dependency'.

Karen logs
----------

tail: cannot open 'logs/karen.log' for reading: No such file or directory

Docker containers
-----------------
NAMES     STATUS

Umbrel logs
-----------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements

Bitcoin Core logs
-----------------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements

LND logs
--------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements

electrs logs
--------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements

Tor logs
--------

ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.bitcoin.ports contains an invalid type, it should be a number, or an object
services.electrs.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.lnd.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
services.lnd.ports value ['9735:9735', ':', ':'] has non-unique elements
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
================
==== Result ====
================
The debug script did not automatically detect any issues with your Umbrel.

```

**Comment by louneskmt** [comment]
Thanks! Is this a new setup?

**Comment by nfaltir** [comment]
No, the power cord was unplugged for some reason when I checked it today. The last time I checked was Mar 1.  I think it was offline all this time :(

**Comment by luisvernet** [comment]
Hey there! Curious if you were able to sort this out? I'm experiencing the same issue but not sure if your issue was that it was unplugged?

**Comment by OpenWaterSwim** [comment]
I built and started running several Umbrel nodes about a year ago. I recently had the power go out. One of my nodes now shows **Error: Failed to start containers** on the browser login screen. I am able to ssh into the node and receive the following errors in the debug log. I had active lightning channels up, so I prefer to fix the container issue than rebuild, but do not know how I can get the containers working again.

**Docker containers**

Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

**Umbrel logs**

ERROR: The Compose file ‘./docker-compose.yml’ is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.auth.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object
Tor logs

ERROR: The Compose file ‘./docker-compose.yml’ is invalid because:
networks.default.ipam.config.subnet is invalid: should use the CIDR format
services.tor_proxy.ports contains an invalid type, it should be a number, or an object
services.auth.ports contains an invalid type, it should be a number, or an object
services.nginx.ports contains an invalid type, it should be a number, or an object

These issues seem to keep me from stopping and starting the docker daemon and service. My research into the errors show quite a few other Umbrel users experiencing the same or similar issues.

**Comment by cdmoss** [comment]
I may have found a fix for this, which I propose in #1558. The docker-compose.yml file located in the root umbrel directory is missing curly braces around the NETWORK_IP environment variable in the `subnet` property at the bottom of the file. Change the file from this: 
![image](https://user-images.githubusercontent.com/40777533/203647579-0bbedcfd-3d09-4706-ba71-10bf3b896791.png)

to this:
![image](https://user-images.githubusercontent.com/40777533/203647668-08d64b8c-9272-457a-8943-058a19abb03f.png).

**Comment by nmfretz** [comment]
Closing this issue. See related https://github.com/getumbrel/umbrel/issues/1558. Please reach out on our [Community Forum](https://community.getumbrel.com/), [Telegram](https://t.me/getumbrel), or via the help chat on our [website](https://umbrel.com/) if you are still having issues or run in to this again.
